### PR TITLE
Feature/french translation

### PR DESCRIPTION
**Pull request by pjgueno**

### CODE REVIEW

File: content/airrohr/fr/00-introduction.md [hunk 1]
```diff
@@ -1,22 +1,32 @@
 ---
 title: Introduction
```
Comment (ohheyitsdave) [inline content/airrohr/fr/00-introduction.md:2]: can you translate the title **introduction**?

File: content/airrohr/fr/03-configure.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+![](../docs/airrohr_config_initial.png)
+<br>
+
+> ⚠️ **Remarque**  La connexion au réseau local peut prendre un peu de temps. Ne soyez pas impatient et essayez plusieurs fois si besoin. La station peut aussi être configurée à l'aide d'un smartphone. Si la configuration a fonctionné, la page décrite ci-dessus ne sera plus accessible directement avec l'adresse IP 192.168.4.1.
+
+### Tester la station
+Si aucun changement particulier n'a été effectué (autre que la saisie du réseau Wifi) lors de la configuration, vous devriez pouvoir consulter les pages test suivantes après environ 10 minutes. Recherchez simplement l'ID de votre station (par exemple 13597771).
+
+ * [Sensordaten](www.madavi.de/sensor/graph.php) 
+ * [WLAN-Signal](www.madavi.de/sensor/signal.php)
```
Comment (ohheyitsdave) [inline content/airrohr/fr/03-configure.md]: can you translate **Sensordaten** and **WLAN-Singal**?

**Review by ohheyitsdave — CHANGES_REQUESTED** [review]
Hi @pjgueno,
I found one small thing in 03-configure.md. Rest looks fine 😊

**Comment by pjgueno** [comment]
Introduction = Introduction in French
The rest should be fine. I translated the 2 words.

**Review by ohheyitsdave — APPROVED** [review]
thank you, @pjgueno

**Pull request merged**